expansion/src/test/java/no/unit/nva/expansion/ResourceExpansionServiceTest.java

```diff
@@ -670,7 +670,7 @@ void shouldExpandPublicationWithNviStatusWhenPublicationIsReportedNviCandidate()
         var nviStatusNode = json.get(ScientificIndex.SCIENTIFIC_INDEX_FIELD);
 
         assertThat(nviStatusNode.get("year").asText(), is(equalTo("2024")));
-        assertThat(nviStatusNode.get("status").asText(), is(equalTo("Reported")));
+        assertThat(nviStatusNode.get("status").asText(), is(equalTo("REPORTED")));
```

Comment:
@truhacevkir Might this be a problem for `search-api`?

Reply:
I dont think so, I mean search does simple aggregation for scientific index statuses, frontend on other side should be aware of this change.

Reply:
Changed json serialization, so it should be serialized "Reported" now